packages/@sfcc-core/apiconfig/package.json

```diff
@@ -1,20 +1,44 @@
 {
-  "name": "@sfcc-core/apiconfig",
-  "version": "1.0.0-alpha.1",
-  "description": "sfcc backend for frontend api service",
-  "main": "./src/apiconfig.js",
-  "type": "module",
+  "license": "BSD-3-Clause",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "lint": "eslint src",
-    "lint:fix": "eslint src --fix",
-    "pak": "npm pack",
-    "pub": "npm pack; npm publish -f",
-    "link": "npm link",
-    "unlink": "npm link"
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test:unit": "tsdx test --collect-coverage",
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
+    "pub": "npm build; npm pack; npm publish -f",
+    "pack": "npm pack"
   },
-  "author": "Jason Cooper",
-  "license": "MIT",
+  "peerDependencies": {},
   "dependencies": {
     "@sfcc-core/core": "1.0.0-alpha.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "tsdx lint"
+    }
+  },
+  "name": "@sfcc-core/apiconfig",
+  "version": "1.0.0-alpha.1",
+  "description": "sfcc backend for frontend api service",
+  "module": "dist/apiconfig.esm.js",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "default": "./dist/apiconfig.esm.js"
```

Comment:
The package is meant to support both native ESM builds, as well as older commonjs